### ISSUE TITLE
Why could you pretrain your model through DDPM, but sample images using DAPS through EDM sampler?

### ISSUE THREAD

**Issue by championOC** [zhangbingliang2019/DAPS#11]
For example, their exists 2 tasks examples, [ffhq-256] and [imagenet-256]，their pretrained checkpoint are based on DDPM (guided ddpm). But in DAPS, you can use these checkpoint and get image samples through EDM ,why could this phenomena achieved?

**Comment by zhangbingliang2019** [comment]
As mentioned in the paper, we apply the preconditioning introduced in [EDM](https://arxiv.org/abs/2206.00364) paper only for inference. Surprisingly, all models tested (see README) perform comparable or even better in the EDM scheduler than the training scheduler. 

For implementation, you can refer to `cores.scheduler` for different diffusion schedulers (VP, VE, EDM, etc.) and `model.__init__.DiffusionModel` for a unified diffusion interface. Theoretically, you can use any scheduler in DAPS.

**Comment by championOC** [comment]
Thank you !



---Original---
From: ***@***.***&gt;
Date: Tue, Mar 25, 2025 10:35 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [zhangbingliang2019/DAPS] Why could you pretrain your modelthrough DDPM, but sample images using DAPS through EDM sampler? (Issue #11)


   
As mentioned in the paper, we apply the preconditioning introduced in EDM paper only for inference. Surprisingly, all models tested (see README) perform comparable or even better in the EDM scheduler than the training scheduler.
 
For implementation, you can refer to cores.scheduler for different diffusion schedulers (VP, VE, EDM, etc.) and model.__init__.DiffusionModel for a unified diffusion interface. Theoretically, you can use any scheduler in DAPS.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
  zhangbingliang2019 left a comment (zhangbingliang2019/DAPS#11)
 
As mentioned in the paper, we apply the preconditioning introduced in EDM paper only for inference. Surprisingly, all models tested (see README) perform comparable or even better in the EDM scheduler than the training scheduler.
 
For implementation, you can refer to cores.scheduler for different diffusion schedulers (VP, VE, EDM, etc.) and model.__init__.DiffusionModel for a unified diffusion interface. Theoretically, you can use any scheduler in DAPS.

**Comment by xuyangth** [comment]
Why your model use sigam as the condition variable not t ? How is your model trained

**Comment by zhangbingliang2019** [comment]
@xuyangth Please refer to [our response](https://github.com/zhangbingliang2019/DAPS/issues/12) in another issue.
